### DISCUSSION TITLE
Would you use Golang support if it was added to Deta Micros?

### DISCUSSION THREAD

**Discussion by s4m-mo** [deta/cloud-docs#574] [Polls]
Would you use Golang support if it was added to Deta Micros?

**Comment by abdelhai** [comment]
hi Sam, than you for your input. we have golang support on deta space (our new platform), apply for access here: https://alpha.deta.space/

**Reply by gitcfly** [reply]
Thank you for your help, because golang is not supported in deta, so I switched to space, but found that custom subdomains are not supported in space, but the default address is too difficult to remember, the function of this custom subdomain can be Let go in space? Like in deta.
